tailwind.config.js

```diff
@@ -90,6 +128,17 @@ module.exports = {
         introToRight2: "introToRight 0.5s cubic-bezier(0.175, 0.885, 0.320, 1.275) 0.5s both",
         introToLeft1: "introToLeft 0.5s cubic-bezier(0.175, 0.885, 0.320, 1.275)",
         introToLeft2: "introToLeft 0.5s cubic-bezier(0.175, 0.885, 0.320, 1.275) 0.5s both",
+        navModalFadeIn: "navModalFadeIn 0.3s ease-in-out",
+        navModalFadeOut: "navModalFadeOut 0.3s ease-in-out",
+      },
+      screens: {
+        tablet: "834px",
+        desktop: "1280px",
```

Comment:
Please use the default responsive breakpoint instead of customising them.

https://tailwindcss.com/docs/responsive-design